### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by modelonrobinandersson**
Closes https://github.com/modelon-community/PyFMI/issues/202

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +29,8 @@ Installation using CONDA
 Note that some examples requires optional dependencies for plotting, they are:
 - wxPython
 - matplotlib
+
+FMU Import Compliance
+-----------
+PyFMI is tested on an everyday basis both with the unit tests found in the `tests` directory but also extensively outside of this project with the `Optimica Compiler Toolkit` (OCT) provided by [Modelon](https://help.modelon.com/latest/reference/oct/). All models from the libraries provided by Modelon have been tested via simulation tests (which covers over 30000+ Modelica models) as FMUs (using the FMU Export functionality from OCT). PyFMI is the default execution engine for [Modelon Impact](https://modelon.com/modelon-impact/).
```
Comment (PeterMeisrimelModelon) [inline README.md:35]: PyFMI contains a number of FMUs used in these unit tests, we should some references to the generating tool for these.

Comment (modelonrobinandersson) [reply README.md]: Made some updates, see my latest commit.

Comment (PeterMeisrimelModelon) [reply README.md]: This is valid for the examples/files/FMUs, but not for the ones in test/*, these do not have any ReadMe. Aside from that, I think there would still be value in having this information directly accessible here.

Also, thoughts on maybe formatting this via some brief bullets? I.e., making the text more factual and less conversational?

File: README.md [hunk 2]
```diff
@@ -32,3 +29,14 @@ Installation using CONDA
 Note that some examples requires optional dependencies for plotting, they are:
 - wxPython
 - matplotlib
+
+FMU Import Compliance
+-----------
+PyFMI is tested daily in several different ways:
+* Extensive external testing within the `Optimica Compiler Toolkit` (OCT) provided by [Modelon](https://help.modelon.com/latest/reference/oct/) using all models in Modelon Modelica libararies (35000+ models), exported as FMUs.
+* Unit tests from the `tests` directory. These use FMUs generated by
+* * OCT
```
Comment (PeterMeisrimelModelon) [inline README.md:38]: Only one * ? Looks a bit funny in the preview

**Review by PeterMeisrimelModelon — APPROVED** [review]

**Pull request merged**